ui/pages/confirmations/components/confirm/info/hooks/useFeeCalculations.ts

```diff
@@ -32,14 +32,18 @@ export function useFeeCalculations(transactionMeta: TransactionMeta) {
   const conversionRate = useSelector(getConversionRate);
   const fiatFormatter = useFiatFormatter();
 
+  const multichainNetwork = useSelector(getMultichainNetwork);
```

Comment:
This is okay in the context of a fix, but long-term this is ultimately still using `getProviderConfig` where as we'll want to pass in our transaction chain ID somehow.